modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -291,8 +312,9 @@ namespace {
         // [[codegen::verbatim(ColorUseABlendingInfo.description)]]
         std::optional<bool> alphaBlendingEnabled;
 
-        // Set if first/last file should render forever.
-        bool showAtAllTimes;
+        // Set if first/last file should render when time is outside of the sequence
+        // interval. This can be used regardless of LoadingType.
```

Comment:
```suggestion
        // interval. This can be used regardless of LoadingType. If this value is not
        // specified, the field lines are shown at all times.
```